test/EFCore.Analyzers.Tests/InterpolatedStringUsageInRawQueriesTests.cs

```diff
@@ -0,0 +1,683 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.EntityFrameworkCore;
+
+using VerifyCS = CSharpCodeFixVerifier<InterpolatedStringUsageInRawQueriesDiagnosticAnalyzer, InterpolatedStringUsageInRawQueriesCodeFixProvider>;
+
+public class InterpolatedStringUsageInRawQueriesTests
+{
+    private const string MyDbContext = """
+using Microsoft.EntityFrameworkCore;
```

Comment:
FYI Our current style is to put the opening `"""` on the opening line, as follows. Although our current automated style fixer unfortunately doesn't support this... Feel free to do it either way.

```suggestion
    private const string MyDbContext =
"""
using Microsoft.EntityFrameworkCore;
```